### ISSUE TITLE
Unable to view bottom of images with alt text when zoomed in

### ISSUE THREAD

**Issue by ericlathrop** [pachli/pachli-android#337]
**Describe the bug**
If an image has alt text and you zoom in, you cannot we the bottom of the image because its covered by tye alg text

**To Reproduce**
1. Find a toot with an image
2. Click on the image to view it full screen
3. Zoom in on the image
4. Notice that the alt text is covering the bottom of the image.

**Expected behavior**
I should be able to pan the image upwards to see the bottom of the image, or hide the alt text.

**Screenshots or video**
![Screenshot_20231227-123128](https://github.com/pachli/pachli-android/assets/530891/b45530f2-a956-4bcf-b252-924cc07a3ad4)

**Links**
The issue is demonstrated on this post: https://sfba.social/@studiochris/111651977925585230

**Versions**
Pachli 2.0.0
Android 14
Mastodon 4.2.3

**Affirmation**
I have checked other issues for this project (open and closed) and I cannot find one that matches the problem I am reporting.

**Comment by nikclayton** [comment]
I did the following on the same post:

1.  Find a toot with an image
2.  Click on the image to view it full screen
3.  Zoom in on the image
4.  Notice that the alt text is covering the bottom of the image.
5. **Tap the zoomed in image to hide the alt text (it also hides the toolbar UI at the top)**. You can tap the image again to show the alt text and other UI. This works whether or not you are zoomed in.

This is also the behaviour if you're not zoomed in -- default is to show the alt text and UI, but you can tap to hide it, tap again to show.

**Comment by ericlathrop** [comment]
Ah, cool, I didn't know that worked. I was trying to drag the drag handle on the alt text, and it doesn't collapse all the way.
